README.md

```diff
@@ -1,3 +1,9 @@
+# david-cako/react-native-maps
```

Comment:
Please change the readme  only with the functionality (lower in the examples or where their place is)